packages/block/src/header/header.ts

```diff
@@ -560,15 +560,32 @@ export class BlockHeader {
    * Calculates the excess blob gas for next (hopefully) post EIP 4844 block.
    */
   public calcNextExcessBlobGas(childCommon: Common): bigint {
-    // The validation of the fields and 4844 activation is already taken care in BlockHeader constructor
-    const targetGasConsumed = (this.excessBlobGas ?? BIGINT_0) + (this.blobGasUsed ?? BIGINT_0)
-    const targetBlobGasPerBlock = childCommon.param('targetBlobGasPerBlock')
+    const excessBlobGas = this.excessBlobGas ?? 0n
```

Comment:
Please do not use the `n` notation for BigInts (compatibility reasons) + we have these predefined constants (like `BIGINT_1`) for various commonly used BigInt values, which gives a performance benefit and which you can use on various occasions in this PR.